cnrancher/edge-api-server
# [WIP] update vendor and integration test env config

**Problem:**
1. integration test will be failed if the user's env contains the proxy config.
2. api-server will return `fatal error: concurrent map read and map write` when applying a new CRD use the `kubectl apply -f devices.edge.cattle.io_bluetoothdevices.yaml`

**Solution:**
1. unset proxy env config in the test script.
2. set sync lock to the lookupSchema since it will trigger concurrent map read action.

**Test Plan:**
1. run `./script/test` on the local machine with the proxy config enabled and ensure it will still succeed.
2. run `kubectl apply/delete -f devices.edge.cattle.io_bluetoothdevices.yaml` for multiple times and ensure it will not cause `fatal error: concurrent map read and map write`  error in the api-server.
